app/views/channels.html

```diff
@@ -50,14 +50,35 @@
                     <span ng-if="!channel.priority">-</span>
                   </td>
                   <td data-title="Channel Name">
-                    <a ng-if="channel.status!='deleted'" href="" ng-click="editChannel(channel)">{{ channel.name }}</a>
+                    <a tooltip={{channel.description}} ng-if="channel.status!='deleted'" href="" ng-click="editChannel(channel)">{{ channel.name }}</a>
                     <span ng-if="channel.status=='deleted'">{{ channel.name }}</span>
                   </td>
                   <td data-title="Channel URL Pattern">{{ channel.urlPattern }}</td>
-                  <td data-title="Description">
-                    <a ng-if="channel.status!='deleted'" href="" ng-click="editChannel(channel)">{{ channel.description }}</a>
-                    <span ng-if="channel.status=='deleted'">{{ channel.description }}</span>
+                  
+                  <td data-title="Access">
+                    <div class="channelAccessControlDiv" ng-if="channel.status!='deleted'" href="" ng-click="editChannel(channel, 'Access Control')">
```

Comment:
The access control tab isn't the right one for public and whitelist. I think this has been changed around in the latest in master, so, perhaps merge in master and revise this.